src/debugger/direct/directDebugAdapter.ts

```diff
@@ -69,30 +69,34 @@ export class DirectDebugAdapter extends ChromeDebugAdapter {
 
         return new Promise<void>((resolve, reject) => this.initializeSettings(launchArgs)
             .then(() => {
-                this.outputLogger("Launching the app");
-                logger.verbose(`Launching the app: ${JSON.stringify(launchArgs, null , 2)}`);
-                return TelemetryHelper.generate("launch", extProps, (generator) => {
-                    return this.remoteExtension.launch({ "arguments": launchArgs })
-                        .then(() => {
-                            return this.remoteExtension.getPackagerPort(launchArgs.cwd);
+                this.outputLogger("Launching the application");
+                logger.verbose(`Launching the application: ${JSON.stringify(launchArgs, null , 2)}`);
+                return ReactNativeProjectHelper.getReactNativeVersionFromProjectPackage(launchArgs.cwd)
```

Comment:
I replaced
```
return ReactNativeProjectHelper.getReactNativeVersionFromProjectPackage(launchArgs.cwd)
```
with
```
return ReactNativeProjectHelper.getReactNativeVersion(attachArgs.cwd)
```
I did so, because there was an error handling problem when 
```
return ReactNativeProjectHelper.getReactNativePackageVersionFromNodeModules(launchArgs.cwd)
```
function was used here.